app/models/katello/subscription.rb

```diff
@@ -33,6 +37,14 @@ def recently_expired?
       pools.any?(&:recently_expired?)
     end
 
+    def virt_who
```

Comment:
i find the class and instance methods on the same name confusing, is it on purpose ?